lib/puma/binder.rb

```diff
@@ -275,9 +282,6 @@ def add_ssl_listener(host, port, ctx,
     end
 
     def inherit_ssl_listener(fd, ctx)
-      require 'puma/minissl'
```

Comment:
Shouldn't we still have the `raise` here, like in `add_ssl_listener`?

Reply:
I can add it.  I thought that `inherit_ssl_listener` would never be called unless on a restart, but maybe with systemd?

Reply:
I added it.